specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/examples/ManagedHsm_CreateOrUpdate.json

```diff
@@ -0,0 +1,88 @@
+{
+  "parameters": {
+    "resourceGroupName": "hsm-group",
+    "name": "hsm1",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2020-04-01-preview",
+    "parameters": {
+      "properties": {
+        "tenantId": "00000000-0000-0000-0000-000000000000",
+        "initialAdminObjectIds": [
+          "00000000-0000-0000-0000-000000000000"
+        ],
+        "enableSoftDelete": true,
```

Comment:
We should not give an option to set enableSoftDelete to false right? So why should we include this parameters for create/update of MHSM resource? @amitbapat can confirm.

Reply:
Discussed with Ritesh. We are doing this because same object is used in the body of the PUT and in the response of the GET.